tools/protodoc/protodoc.bzl

```diff
@@ -82,12 +82,12 @@ proto_doc_aspect = aspect(
         "_protoc": attr.label(
             default = Label("@com_google_protobuf//:protoc"),
             executable = True,
-            cfg = "host",
+            cfg = "exec",
```

Comment:
what is `exec`? shouldn't it be `host` or `target`? https://docs.bazel.build/versions/master/skylark/lib/attr.html#label

Reply:
Looks like the docs haven't been updated, but exec is a third configuration for the "execution environment" which in this case is RBE, not your host machine (as far as I understand it) https://github.com/bazelbuild/proposals/blob/master/designs/2019-02-12-execution-transitions.md#proposal

Reply:
Ah! I didn't notice that. Then this looks reasonable.

Reply:
I am kind of surprised through your RBE tests that this wasn't a problem, but maybe if host matched exec because of the docker container this was fine

Reply:
I didn't use RBE for docs, because it is fast enough to run locally.